pkg/virt-config/config_test.go

```diff
@@ -482,4 +482,38 @@ var _ = Describe("ConfigMap", func() {
 		emulation = clusterConfig.IsUseEmulation()
 		Expect(emulation).To(BeFalse())
 	})
+
+	It("when feature-gate SRIOVLiveMigration is on, LiveMigrationEnabled should return true", func() {
+		clusterConfig, _, _, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
+			Data: map[string]string{virtconfig.FeatureGatesKey: "SRIOVLiveMigration"},
+		})
+		Expect(clusterConfig.SRIOVLiveMigrationEnabled()).To(BeTrue())
+
```

Comment:
Newline should probably be there between the setup of the test (creating the fake) and the asserts. But I see no reason for it between the asserts.

Reply:
I changed those tests to DescribeTable